azure-pipelines.yml

```diff
@@ -0,0 +1,109 @@
+variables:
+  PreferredToolArchitecture: x64
```

Comment:
Nitpick:  `PreferredToolArchitecture` is unused. I guess you might as well remove it.

Reply:
Are you sure it is unused? How do you know?